process/transaction/shardProcess.go

```diff
@@ -539,10 +539,6 @@ func (txProc *txProcessor) processMoveBalance(
 		txProc.txFeeHandler.ProcessTransactionFee(moveBalanceCost, big.NewInt(0), txHash)
 	}
 
-	if len(tx.RelayerAddr) > 0 {
```

Comment:
Can you remind me why is this no longer needed?
I remember discussing about this, but don't remember exactly the reason.

Reply:
this is not needed as now we'll only use the exact gas needed for move balance, thus no refund will happen. Also, this was wrong here, it would apply for all inner txs, not move balance only as it was supposed to be